bdai_ros2_wrappers/bdai_ros2_wrappers/logging.py

```diff
@@ -37,6 +44,229 @@
         warnings.warn("TODO: use subloggers in RcutilsLogHandler implementation", stacklevel=1)
 
 
+@functools.lru_cache(maxsize=1024)
+def make_logging_function(
+    name: str,
+    level: LoggingSeverity,
+    origin: inspect.Traceback,
+    throttle_duration_sec: typing.Optional[float] = None,
+    throttle_time_source: typing.Optional[Clock] = None,
+    skip_first: typing.Optional[bool] = None,
+    once: typing.Optional[bool] = None,
```

Comment:
What is the difference between `None` and `False`?

Reply:
In this context, none. I annotate optional arguments as optional purely out of habit (and for the sake of readability).

Reply:
Ok, not going to block on this.

I think whether `bool = False` or `Optional[bool] = None` is more readable is another discussion lol.